### PR TITLE
Score summarization frontend

### PR DESCRIPTION
**Pull request by mehalshah**
image

### CODE REVIEW

File: apps/src/code-studio/pd/application_dashboard/detail_view.jsx [hunk 1]
```diff
@@ -41,6 +41,10 @@ export default class DetailView extends React.Component {
     });
   }
 
+  componentWillMount() {
+    this.load();
+  }
+
```
Comment (aoby) [inline apps/src/code-studio/pd/application_dashboard/detail_view.jsx]: nit: move `componentWillMount` up before `load`, so all the standard initialization methods are together.

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx [hunk 2]
```diff
@@ -38,10 +38,12 @@ export default class DetailViewContents extends React.Component {
...
       application_type: PropTypes.oneOf(['Facilitator', 'Teacher']),
-      response_scores: PropTypes.object
+      response_scores: PropTypes.object,
+      meets_criteria: PropTypes.string,
+      bonus_points: PropTypes.number
     }),
-    updateProps: PropTypes.func.isRequired,
-    viewType: PropTypes.oneOf(['teacher', 'facilitator']).isRequired
+    viewType: PropTypes.oneOf(['teacher', 'facilitator']).isRequired,
+    loadFunction: PropTypes.func.isRequired
```
Comment (aoby) [inline apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx]: nit: This reads like it's going to load a function. How about something like `onRequestData` or `onReload`?

File: dashboard/app/models/pd/application/teacher1819_application.rb [hunk 3]
```diff
@@ -400,5 +402,33 @@ def principal_approval_url
...
+      responses = scored_questions.map do |key|
+        response_scores[key]
+      end
+
+      if responses.uniq == [YES]
+        # If all resolve to Yes, applicant meets criteria
+        'Yes'
+      elsif responses.include? NO
+        # If any are No, applicant does not meet criteria
+        'No'
```
Comment (aoby) [inline dashboard/app/models/pd/application/teacher1819_application.rb]: These strings are the same as the constant values for `YES` and `NO`. Why not use those?

File: dashboard/app/models/pd/application/teacher1819_application.rb [hunk 4]
```diff
@@ -400,5 +402,33 @@ def principal_approval_url
...
+      else
+        'Incomplete'
```
Comment (aoby) [inline dashboard/app/models/pd/application/teacher1819_application.rb]: similarly, perhaps this should be a constant.

File: apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx (hunk 2 above)
Comment (mehalshah) [reply apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx]: I'm not sure about either of those names - this function is supposed to reload the data (without displaying the spinner) but I see your point. Let me think

**Pull request merged**